ui/app/components/pages/first-time-flow/create-password/new-account/new-account.component.js

```diff
@@ -111,12 +112,29 @@ export default class NewAccount extends PureComponent {
     history.push(INITIALIZE_IMPORT_WITH_SEED_PHRASE_ROUTE)
   }
 
+  toggleTermsCheck = () => {
+    this.setState({
```

Comment:
We should prefer [the `updater` function argument](https://reactjs.org/docs/react-component.html#setstate) here. That is,

```js
this.setState((prevState) => ({
    termsChecked: !prevState.termsChecked,
})
```

Reply:
Fixed in 51fb72b40